react/features/base/conference/actions.js

```diff
@@ -199,7 +199,8 @@ export function conferenceFailed(conference: Object, error: string) {
         // Make the error resemble an Error instance (to the extent that
         // jitsi-meet needs it).
         error: {
-            name: error
+            name: error,
+            recoverable: undefined
```

Comment:
why do we need this? if an unknown property is checked it will return undefined anyway.

Reply:
flow was complaining about missing property and it silenced it